config.w32

```diff
@@ -140,6 +151,14 @@ if (PHP_MONGODB != "no") {
     WARNING("mongodb libsasl support not enabled, libs not found");
   }
 
+  if (typeof COMPILER_NAME === 'string') {
+    mongoc_opts.MONGOC_CC = COMPILER_NAME;
+  } else if (typeof VC_VERSIONS[VCVERS] === 'string') {
+    mongoc_opts.MONGOC_CC = VC_VERSIONS[VCVERS];
```

Comment:
* [VC_VERSIONS in PHP 7.0](https://github.com/php/php-src/blob/php-7.0.10/win32/build/confutils.js#L56)
 * [VC_VERSIONS in PHP 5.x](https://github.com/php/php-src/blob/php-5.6.25/win32/build/confutils.js#L43)

They are consistent for VC, but PHP 7.0 also supports CLANG and ICC. `COMPILER_NAME` is initialized from [`toolset_get_compiler_name()`](https://github.com/php/php-src/blob/php-7.0.10/win32/build/confutils.js#L2583) if you'd like to see the logic.